Description: fix an error in CMake sequence, avoiding a FTBFS. (Closes: #808760)
Author: Joao Eriberto Mota Filho <eriberto@debian.org>
Last-Update: 2016-03-18
Index: kdbg-2.5.4/CMakeLists.txt
===================================================================
--- kdbg-2.5.4.orig/CMakeLists.txt
+++ kdbg-2.5.4/CMakeLists.txt
@@ -4,7 +4,7 @@ set(KDBG_VERSION 2.5.4)
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/kdbg/version.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/kdbg/version.h)
 
 find_package(KDE4 REQUIRED)
-
+include(CheckIncludeFiles)
 add_definitions(${QT_DEFINITIONS} ${KDE4_DEFINITIONS})
 
 include(KDE4Defaults)
